### PR TITLE
Add routing-api job to api instance group

### PR DESCRIPTION
**Pull request by f0rmiga**
## Description

Added routing-api, locket and doppler. It also uses the loggregator addons.

## Test plan

The listed jobs above should come up. (router is still failing and will be handled on a separate PR)

### CODE REVIEW

**Comment by andreas-kupries** [comment]
> Seems generally reasonable, though I'm not comfortable enough with the code base to definitively approve (I'd like Andreas to take a look too).

You mean Vlad for sure, right ? (I am in your boat too ;))

File: bosh/releases/pre_render_scripts/api/routing-api/patch_bpm.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+patch --binary --unified /var/vcap/all-releases/jobs-src/routing/routing-api/templates/bpm.yml.erb <<'EOT'
```
[the remark this answers is not in the document]
Comment (andreas-kupries) [reply bosh/releases/pre_render_scripts/api/routing-api/patch_bpm.sh]: Actually his patch data is just chunks, without the leading lines (`+++ ...` and `--- ...`) patch AFAIK uses to detect the format.

[the remark this answers is not in the document]
Comment (f0rmiga) [reply bosh/releases/pre_render_scripts/api/routing-api/patch_bpm.sh]: Yeah. I'll leave as is.

**Comment by andreas-kupries** [comment]
With the operator pointing toi a useful commit all the initialization is ok. However both api and router pods are failing ...

```
cf                  scf-dev-api-v1-0                            8/9     CrashLoopBackOff   8          9m17s
scf                  scf-dev-api-v1-1                            8/9     CrashLoopBackOff   5          5m47s
scf                  scf-dev-database-v1-0                       5/5     Running            0          9m19s
scf                  scf-dev-diego-api-v1-0                      5/5     Running            4          9m19s
scf                  scf-dev-diego-api-v1-1                      5/5     Running            4          8m54s
scf                  scf-dev-diego-cell-v1-0                     5/5     Running            0          9m17s
scf                  scf-dev-diego-cell-v1-1                     5/5     Running            0          8m41s
scf                  scf-dev-doppler-v1-0                        8/8     Running            0          9m17s
scf                  scf-dev-doppler-v1-1                        8/8     Running            0          8m36s
scf                  scf-dev-doppler-v1-2                        8/8     Running            0          8m5s
scf                  scf-dev-doppler-v1-3                        8/8     Running            0          7m34s
scf                  scf-dev-log-api-v1-0                        7/7     Running            0          9m16s
scf                  scf-dev-log-api-v1-1                        7/7     Running            0          8m34s
scf                  scf-dev-nats-v1-0                           5/5     Running            0          9m19s
scf                  scf-dev-nats-v1-1                           5/5     Running            0          8m57s
scf                  scf-dev-router-v1-0                         4/5     CrashLoopBackOff   6          9m17s
scf                  scf-dev-router-v1-1                         4/5     CrashLoopBackOff   6          7m14s
scf                  scf-dev-singleton-blobstore-v1-0            6/6     Running            0          9m17s
scf                  scf-dev-uaa-v1-0                            5/5     Running            0          9m18s
scf                  scf-dev-uaa-v1-1                            5/5     Running            0          7m59s
````

... uaa issues ... ok, so that is what the other PR is for. Also issues in the log jobs.

**Comment by f0rmiga** [comment]
> With the operator pointing toi a useful commit all the initialization is ok. However both api and router pods are failing ...
> 
> ```
> cf                  scf-dev-api-v1-0                            8/9     CrashLoopBackOff   8          9m17s
> scf                  scf-dev-api-v1-1                            8/9     CrashLoopBackOff   5          5m47s
> scf                  scf-dev-database-v1-0                       5/5     Running            0          9m19s
> scf                  scf-dev-diego-api-v1-0                      5/5     Running            4          9m19s
> scf                  scf-dev-diego-api-v1-1                      5/5     Running            4          8m54s
> scf                  scf-dev-diego-cell-v1-0                     5/5     Running            0          9m17s
> scf                  scf-dev-diego-cell-v1-1                     5/5     Running            0          8m41s
> scf                  scf-dev-doppler-v1-0                        8/8     Running            0          9m17s
> scf                  scf-dev-doppler-v1-1                        8/8     Running            0          8m36s
> scf                  scf-dev-doppler-v1-2                        8/8     Running            0          8m5s
> scf                  scf-dev-doppler-v1-3                        8/8     Running            0          7m34s
> scf                  scf-dev-log-api-v1-0                        7/7     Running            0          9m16s
> scf                  scf-dev-log-api-v1-1                        7/7     Running            0          8m34s
> scf                  scf-dev-nats-v1-0                           5/5     Running            0          9m19s
> scf                  scf-dev-nats-v1-1                           5/5     Running            0          8m57s
> scf                  scf-dev-router-v1-0                         4/5     CrashLoopBackOff   6          9m17s
> scf                  scf-dev-router-v1-1                         4/5     CrashLoopBackOff   6          7m14s
> scf                  scf-dev-singleton-blobstore-v1-0            6/6     Running            0          9m17s
> scf                  scf-dev-uaa-v1-0                            5/5     Running            0          9m18s
> scf                  scf-dev-uaa-v1-1                            5/5     Running            0          7m59s
> ```
> 
> ... uaa issues ... ok, so that is what the other PR is for. Also issues in the log jobs.

Rebased.

**Comment by andreas-kupries** [comment]
Confirmed

**Pull request merged**